### PR TITLE
Added solar forecast and update DER.

### PR DESCRIPTION
**Pull request by Jeffrey-Simpson**
Added solar forecast and update DER to platform read the docs.

---
This change is [Reviewable](https://reviewable.io/reviews/gridappsd/gridappsd-docs/11)

### CODE REVIEW

File: docs/source/hosted_applications/NREL_APPS/Real_Time_DER_Setpoint_Dispatch.rst [hunk 1]
```diff
@@ -47,32 +47,33 @@ the updated model and measurements.
 its interaction with GridApps=D.**
 
 Data requirements
+~~~~~~~~~~~~~~~~~~~~~~
```
Comment (craig8) [inline docs/source/hosted_applications/NREL_APPS/Real_Time_DER_Setpoint_Dispatch.rst]: headers should only goto the end of the line

eg

Data requirements
~~~~~~~~~~~~

File: docs/source/hosted_applications/NREL_APPS/Solar_Forecasting.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
+Solar Forecasting Application
+---------------------------------------------------------------
```
Comment (craig8) [inline docs/source/hosted_applications/NREL_APPS/Solar_Forecasting.rst]: Headers only to the end of the above line

File: docs/source/hosted_applications/NREL_APPS/Real_Time_DER_Setpoint_Dispatch.rst (hunk 1 above)
Comment (Jeffrey-Simpson) [reply docs/source/hosted_applications/NREL_APPS/Real_Time_DER_Setpoint_Dispatch.rst]: Okay fixed.

File: docs/source/hosted_applications/NREL_APPS/Solar_Forecasting.rst (hunk 2 above)
Comment (Jeffrey-Simpson) [reply docs/source/hosted_applications/NREL_APPS/Solar_Forecasting.rst]: Okay Fixed.

**Pull request merged**